cluster-scope/base/objectbucket.io/objectbucketclaims/acm-metrics/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
```

Comment:
You shouldn't be setting this label here (it should only be set by the top-level kustomization in an overlay).

Reply:
sidetracked:
should we remove this here too? (just a few examples) in a cleaning issue?

- [/cluster-scope/base/objectbucket.io/objectbucketclaims/observability/kustomization.yaml](https://github.com/schwesig/OCP-on-NERC_nerc-ocp-config/blob/d190fa64e3e0e1d8065726c8df81ba777dc60524/cluster-scope/base/objectbucket.io/objectbucketclaims/observability/kustomization.yaml)
- [/cluster-scope/base/observability.open-cluster-management.io/multiclusterobservabilities/observability/kustomization.yaml](https://github.com/schwesig/OCP-on-NERC_nerc-ocp-config/blob/d190fa64e3e0e1d8065726c8df81ba777dc60524/cluster-scope/base/observability.open-cluster-management.io/multiclusterobservabilities/observability/kustomization.yaml)
- [/cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-multiclusterhub-operator-pull-secret/kustomization.yaml](https://github.com/schwesig/OCP-on-NERC_nerc-ocp-config/blob/d190fa64e3e0e1d8065726c8df81ba777dc60524/cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-multiclusterhub-operator-pull-secret/kustomization.yaml)
- [/cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-thanos-object-storage/kustomization.yaml](https://github.com/schwesig/OCP-on-NERC_nerc-ocp-config/blob/d190fa64e3e0e1d8065726c8df81ba777dc60524/cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-thanos-object-storage/kustomization.yaml)
- ...